libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationManager.java

```diff
@@ -30,22 +31,28 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NotificationManager.class);
 
-    private NotificationService notificationService;
+    private List<NotificationService> notificationServices = new ArrayList<>();
     private ScheduledExecutorService scheduledExecutor;
     private List<NotificationTask> notificationTasks;
 
     public NotificationManager(List<NotificationTask> notificationTasks) {
         this.notificationTasks = notificationTasks;
-        String notificationServiceFactoryClass = System.getProperty(NOTIFICATION_PROP_SERVICE_FACTORY_CLASS);
-        if (notificationServiceFactoryClass != null) {
-            NotificationServiceFactory notificationServiceFactory;
-            try {
```

Comment:
Notification Manager can now send notifications to multiple Notification Services.
Users will need to specify the NotificationServiceFactory classes in a comma-separated property.